packages/core/src/components/forms/_common.scss

```diff
@@ -84,6 +84,14 @@ $control-group-stack: (
   }
 }
 
+@mixin pt-input-disabled() {
```

Comment:
Mixins don't need to come before they're declared. Similar to what we do for buttons, I prefer having `pt-input()` first followed by its modifiers `pt-input-<modifier>()`